atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -258,7 +258,7 @@ def group_manifests_and_tag(self, session, worker_digests):
             push_conf_registry.digests[image.tag] = digest
 
         self.log.info("%s: Manifest list digest is %s", session.registry, digest_str)
-        return registry_image.repo, digest
+        return registry_image.get_repo(explicit_namespace=False), digest
```

Comment:
why not just use : registry_image.get_repo(), since that explicit_namespace=False is default ?

Reply:
I've specified `explicit_namespace` here so that later on we could remove all traces of it (when v1 would be retired, as `library/` is v1-only thing). 

I don't mind removing it, but I'd rather keep it so that in the future the behaviour of this won't change if `get_repo` is changed

Reply:
ok